pkg/log/log.go

```diff
@@ -130,7 +132,7 @@ func SetLevel(level zapcore.Level) zapcore.Level {
 // just repeats known information. You should almost always use `ShortError`
 // instead of `zap.Error`, unless the error is no longer propagated upwards.
 func ShortError(err error) zap.Field {
-	if err == nil {
+	if err == nil || errors.Cause(err) == context.Canceled {
```

Comment:
which one is better:
1. update `ShortError`
2. add a new function

@WangXiangUSTC what's your opinion?

Reply:
I think 2 is better

Reply:
Do you mean put `err == nil || errors.Cause(err) == context.Canceled`... into a new function?
@WangXiangUSTC

Reply:
how about adding a new function to judge  whether an error can be filter
```
func FilterError(err error) error {
    if errors.Cause(err) == context.Canceled {
         return nil
    }
    return err
}
```

then we can use `ShortError(FilterError(err))`

Reply:
approve the method from @WangXiangUSTC, but maybe a meaningful name needed?

Reply:
How about `FilterCancelError`?

Reply:
`FilterCancelError` LGTM

Reply:
which one is better:
1. return `error`. the usage would be `log.ShortError(log.FilterCancelError(err))`
2. return `zap.Field`. the usage would be `log.FilterCancel(err)`
@csuzhangxc